src/os/bluestore/BlueStore.cc

```diff
@@ -2003,6 +2005,28 @@ void BlueStore::ExtentMap::update(KeyValueDB::Transaction t,
   }
 }
 
+bid_t BlueStore::ExtentMap::allocate_spanning_blob_id()
```

Comment:
The return variable is a right value, should we make it const? As with const it causes the warning "warning: type qualifiers ignored on function return type".